yang/ietf-schedule.yang

```diff
@@ -16,8 +16,12 @@ module ietf-schedule {
     "WG Web: <https://datatracker.ietf.org/wg/opsawg/>
      WG List: <mailto:opsawg@ietf.org>";
   description
-    "This YANG module defines two properties of an iSchedule
-     object:period of time and recurrence rule.
+    "This YANG module defines two groupings for iCalendar (Internet
```

Comment:
Shouldn't this be schedule rather that calendar?

Reply:
Yes, we are defining a schedule module, but it came from the icalendar(which is the title of RFC5545) data value type definition. So I am still using iCalendar concept here but use the 'schedule' elsewhere. Make sense?

Reply:
OK, thanks.